scout/server/blueprints/cases/templates/cases/utils.html

```diff
@@ -293,8 +293,18 @@ <h5 class="modal-title" id="exampleModalLabel">Beacon submission</h5>
 
   {% if variant.category %}
     {% if variant.category in "str" %}
-      {{ variant.str_repid }} {{ variant.alternative | replace("<", " ") | replace(">", "")}}
-
+      {% if variant.str_repid %}
```

Comment:
This takes into account the various IDs and expansion designations possible for different callers.